clients/java/signalr/src/test/java/HubConnectionTest.java

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
```

Comment:
Is this the way this is usually done in JUnit? C# has an equivalent feature (`using static`) that we generally avoid using and I'd initially suggest we align with that. But if this is how JUnit tests are generally written, I'm fine with that :)

Reply:
https://www.tutorialspoint.com/junit/junit_test_framework.htm seems to be fairly standard.